src/main/java/com/monumental/services/ModelService.java

```diff
@@ -281,6 +281,39 @@ public void delete(List<Integer> ids) throws HibernateException {
         return records;
     }
 
```

Comment:
This is a generic method we can use in the future to make queries with `WHERE` clauses on our tables without having to repeat all of the other junk.